### PR TITLE
Add facilitator images

### PR DESCRIPTION
**Pull request by CannonLock**
I made the images a bit larger then this. Information is populated from the team file, but if the # of facilitators change the image size will have to be tweaked. 

Screen Shot 2022-01-25 at 3 46 46 PM

### CODE REVIEW

**Comment by ChristinaLK** [comment]
I love this!! 🎉 @CannonLock you are the best. :DDDD
I do think @lmichael107 and @rachellombardi should weigh in before we merge.

**Review by ChristinaLK — COMMENTED** [review]
Request from the team: Can we have our job titles listed as well? Lauren thought it might be helpful to have it indicated that she is the lead facilitator. That information is in the yaml file as the "job_title" so hopefully that's easy to add underneath our names?

**Comment by CannonLock** [comment]
@ChristinaLK How about this?

Screen Shot 2022-01-28 at 9 28 42 AM

**Comment by ChristinaLK** [comment]
That looks GREAT. thank you!!

**Review by ChristinaLK — APPROVED** [review]

**Pull request merged**
